react/src/lib/components/GroupTree/components/Settings/DateTimeSlider.test.tsx

```diff
@@ -21,9 +21,10 @@ describe("Test  Date-Time Slider", () => {
             payload: undefined,
             type: "ui/updateCurrentDateTime",
         });
-        expect(testStore.dispatch).toHaveBeenNthCalledWith(2, {
-            payload: "01/01/2001",
-            type: "ui/updateCurrentDateTime",
-        });
+        // XXX disabled
```

Comment:
If this will be added back, please make an issue for it, maybe assign it to Avanti (?). Otherwise, just remove the code.